sourced/ml/transformers/basic.py

```diff
@@ -62,6 +62,22 @@ def __call__(self, head: RDD):
         return head
 
 
+class Repartitioner(Transformer):
+    def __init__(self, partitions: int, **kwargs):
+        super().__init__(**kwargs)
+        self.partitions = partitions
+
+    def __call__(self, head: RDD):
+        return head.repartition(self.partitions)
```

Comment:
How about `coalesce()`? I would add a flag to `__init__` and do `coalesce()` by default.

Reply:
repartition is actually 
```python
return self.coalesce(numPartitions, shuffle=True)
```
So we can add `shuffle` flag, yes, but I think in case of performance it can be not so good idea because what we saw is really skewed partitions after reading siva files. 

I can run some performance tests locally if you want.

Reply:
after repartition, I can see that last jobs finish fast one after another and we do not have a problem waiting while one process will finish.

Reply:
about skewed partitions, see the image in 
https://github.com/src-d/engine/issues/351

Reply:
This may be not good for your specific use case but good for the others, this transformer is universal. Feel free to hardcode `shuffle=True` in repos2coocc but make it false by default in the transformer.